include/drivers/clock_control/stm32_clock_control.h

```diff
@@ -19,10 +19,10 @@ enum {
 	STM32_CLOCK_BUS_AHB1,
 	STM32_CLOCK_BUS_AHB2,
 	STM32_CLOCK_BUS_APB1,
+	STM32_CLOCK_BUS_APB2,
 #ifdef CONFIG_SOC_SERIES_STM32L4X
 	STM32_CLOCK_BUS_APB1_2,
 #endif
-	STM32_CLOCK_BUS_APB2,
 };
 
```

Comment:
So best I could do is #include <arm/st/clocks.h>
I'll change the plan and #include <dt_stm32_clocks.h>